.github/workflows/expo_ci.yml

```
@@ -51,7 +51,7 @@ jobs:
           webhook: ${{ secrets.DISCORD_WEBHOOK }}
           status: ${{ job.status }}
           image: ${{ secrets.SUCCESS_IMAGE }}
-          description: 뀨
+          description: @이명훈 @문혜성
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 54: found character that cannot start any token

(syntax-check)